ocaml/xapi/updateinfo.mli

```diff
@@ -111,6 +111,8 @@ module UpdateInfo : sig
     ; update_type: string
     ; livepatch_guidance: Guidance.t option
     ; livepatches: LivePatch.t list
+    ; issued: Xapi_stdext_date.Date.t
+    ; severity: string
```

Comment:
I believe the possible values of `severity` are able to be enumerated. Then a new data type for this would be better.

Reply:
It would be better to encode severity as an integer or enum. The domain of a string is too unrestricted.

Reply:
I think the idea to add an Enum type in XAPI for `severity` is to check the correctness of the value for `severity` from  updateinfo, although it is rarely possible that the value will be invalid, I agree it is an enhancement to add it.
While for the proposal to "match the severity from String to an integer", currently there are only 2 levels of severity are defined: "High" and "None", it is easy to understand the order of the severity, and I think this match might require an extra step to understand the `severity` returned from XAPI, users might need to be aware of what severity in the updateinfo an integer presents.
So how about just add the `severity` enum type which contains "High" and "None"?

Reply:
I'm fine with adding an enum; I was just offering a way to do this with fewer changes.

Reply:
OK, thank you